src/components/UploadButton/index.tsx

```diff
@@ -0,0 +1,132 @@
+import { getMediaType, useUpload } from '@/contexts';
+import { MediaType } from '@/types';
+import classnames from 'classnames';
+import { FC, HTMLProps, useState } from 'react';
+import { Icon, LoadingIcon } from '../Icon';
+import { useLocales } from '../useLocales';
+import styles from './index.module.less';
+
+interface Props extends Omit<HTMLProps<HTMLDivElement>, 'onError'> {
+  fileName?: string;
+  onDragUpload?: (file: File) => void;
+  onStartUpload?: () => void;
+  onUploaded?: (opts: { fileAddressList: string[] | undefined; whitelistFileName: string }) => void;
+  onError?: (err: Error) => void;
+  disabled?: boolean;
+}
+
+const getMaxSize = (mediaType: MediaType) => {
+  switch (mediaType) {
+    case MediaType.Video:
+    case MediaType.Audio:
+      return 30 * 1024 * 1024;
+    case MediaType.Image:
+    case MediaType.Unknown:
+      return 2 * 1024 * 1024;
+    case MediaType.csv:
```

Comment:
Capitalized first letter for enum key.
```suggestion
    case MediaType.Csv:
```